src/index.html

```diff
@@ -4,9 +4,462 @@
+            </li>
+            <li class="nav__items">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__items">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__items">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="Apple laptop"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+              Product code: 195434
+          </p>
+
+          <div class="stars">
+            <div class="stars__icon">
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+            </div>
+            <div class="stars__count">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">
+              Price:
+            </p>
+            <p class="card__count">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="container">
+        <div class="card">
+          <img
+            class="card__img"
+            src="../src/images/imac.jpeg"
+            alt="Apple laptop"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+              Product code: 195434
+          </p>
+
+          <div class="stars">
+            <div class="stars__icon">
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+              <div class="stars__review"></div>
+            </div>
+            <div class="stars__count">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__text">
+              Price:
+            </p>
+            <p class="card__count">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
```

Comment:
If we have only 2 attributes, then we don't need to move them to a new line, we only need to do this when there are 3 or more attributes, we need to fix it everywhere
```suggestion
          <a class="card__buy" href="#">
            Buy
          </a>
```